package.json

```diff
@@ -112,6 +112,7 @@
         "@playwright/browser-chromium": "1.39.0",
         "@playwright/browser-firefox": "1.39.0",
         "@playwright/browser-webkit": "1.39.0",
+        "@types/axios-token-interceptor": "^0.2.7",
```

Comment:
why do u have the axios dependency added for this ticket?

Reply:
I'll remove this line Kian, thanks